subprojects/core/src/main/java/org/gradle/api/internal/file/CopyActionProcessingStreamAction.java

```diff
@@ -18,8 +18,10 @@
 
 import org.gradle.api.internal.file.copy.FileCopyDetailsInternal;
 
+import java.io.File;
+
 public interface CopyActionProcessingStreamAction {
 
-    void processFile(FileCopyDetailsInternal details);
+    File processFile(FileCopyDetailsInternal details);
```

Comment:
this change seems to be causing plugin failure, e.g. https://github.com/johnrengelman/shadow/blob/master/src/main/groovy/com/github/jengelman/gradle/plugins/shadow/tasks/ShadowCopyAction.groovy#L86

Reply:
Fixed